### DISCUSSION TITLE
Command to get current window size.

### DISCUSSION THREAD

**Discussion by thunderstorm010** [hecrj/iced#1901] [Ideas]
Will implement if I have time.

**Comment by rubenh2905** [comment]
when i'm was exploring examples there was a example called events, it was able to listen all native events and when i'm was resizeing the windows it can get it, i think you can listen for that event and have it on catch

**Comment by thunderstorm010** [comment]
Yep, I know that. The problem is when the user never resizes you don't know
the size. Of course you can just hardcode the default size but that's just
a temporary solution.

rubenh2905 ***@***.***> schrieb am Di., 13. Juni 2023, 11:02:
